_posts/2017-12-11-reponse-to-null-effects.md

```diff
@@ -0,0 +1,198 @@
+results not only from embracing error as normal, but also from learners’ personal belief in their ability to succeed. Creating a 
+workshop in which learners can be successful in both learning and in demonstrating to themselves that they have learned is one piece 
+of this. We spend a good deal of time discussing motivation with our instructors. We explain why saying "it's easy, anyone can do it" 
+is often demotivating. We explore the differences between novice and expert perspectives and coach instructors to be mindful of and 
+to respect the novice experience. We teach instructors to foster a growth mindset in their language and learner interactions. We 
+emphasize that a relaxed, welcoming, and positive workshop experience is one of the most important things we can provide. 
+
+**Build community.** The more people at all levels are able to share what they know, the more efficiently we can distribute knowledge. 
+As a volunteer organization, we have a strong community of instructors, lesson maintainers, and others. As learners progress, 
+they often become involved in this community. In the long range, we hope to create a community that can provide widespread 
+support directly to learners. 
+
+#### What we know about our impact
+
+We have conducted both short-term and long-term follow-up assessments of learners. Data Carpentry [post-workshop survey results](https://carpentries.github.io/assessment/data-carpentry/postworkshop/report.html)  have always been positive and 85% of learners report that they agree that they would recommend our workshops to a colleague. The Carpentries’ Long-Term Impact survey (n = 530) is designed to determine whether this positive experience and self-reported increase in confidence affects long term outcomes. This survey ([full report here](https://carpentries.github.io/assessment/carpentries/long-term-survey/report.html)) measured self-reported behaviors around good data management practices, change in confidence in open source tools, and other specific program goals. It also explored other ways the workshop may have impacted learners, such as improved research productivity. While Feldon et al. rightly critique self-assessment with regard to performance metrics, many of our target outcomes are more conducive to self-evaluation, e.g. confidence, motivation, and daily work habits. Researchers report increased daily programming usage after attending our two-day coding workshops, and sixty-five percent of respondents report higher confidence in working with data and open source tools as a result of completing the workshop. Our long-term assessment data shows a decline in the percentage of respondents that 'have not been using these tools' (-11.1%), and an increase in the percentage of those who now use the tools on daily basis (14.5%). Additional highlights from our long-term survey report include:
```

Comment:
It looks like one or more of the links in this section may not render properly, if the color-coding is indicative